client.py

```diff
@@ -110,6 +131,15 @@ def main():
     else:
         logging.basicConfig(level=logging.INFO,format='%(asctime)s %(levelname)s %(message)s')
 
+    # Checking on startd logging configuration if enabled
+    if 'send_startd_logs' in config_startd_logging and config_startd_logging['send_startd_logs'] is True:
+        for config_val in ['url', 'bucket', 'access_key', 'secret_key']:
+            if config_val not in config_startd_logging:
+                logger.error('Missing %s configuration value in StartdLogging Section' % config_val)
+                sys.exit(1)
+        # Creating Bucket for Grid Site
+        create_bucket(config_startd_logging)
```

Comment:
I'm unsure if I like this syntax, where magically `config_startd_logging` gets modified without us really knowing on this end.  It's generally better to return something from the function, so you see the action happening.

Reply:
Reviewed this - it's read-only.  Carry on.